<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>840</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>840</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>840</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MyOpenGLWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>800</width>
      <height>800</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenFileButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>40</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FileNamelineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="RotateBox">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>190</y>
      <width>131</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Rotate</string>
    </property>
    <widget class="QPushButton" name="RotateButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotate</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="RotatelineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="XRotateRB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="YRotateRB">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="ZRotateRB">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="MoveBox">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>190</y>
      <width>131</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Move</string>
    </property>
    <widget class="QLineEdit" name="MovelineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="XMoveRB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="YMoveRB">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="ZMoveRB">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QPushButton" name="MoveButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Move</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>330</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Color</string>
    </property>
    <widget class="QPushButton" name="BackColor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>Background</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LineColor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>Lines</string>
     </property>
    </widget>
    <widget class="QPushButton" name="VerticesColor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Vertices</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>330</y>
      <width>131</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Scale</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>111</width>
       <height>32</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="ScalePlus">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ScaleMinus">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="ScalelineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>69</y>
      <width>271</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Model Info</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>File name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Vertices count</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Edges count</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="FileNameInfLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="VertCountLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="EdgeCountLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>480</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Projection</string>
    </property>
    <widget class="QPushButton" name="ProjectionButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Switch</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>440</y>
      <width>131</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Vertices</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="SquareVert">
          <property name="text">
           <string>Square</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="CircleVert">
          <property name="text">
           <string>Circle</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="VertPlus">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="VertMinus">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="DisablePoints">
        <property name="text">
         <string>Enable/Disable</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>560</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Dashed line</string>
    </property>
    <widget class="QPushButton" name="DashedLine">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable/Disable</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>630</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Line width</string>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>32</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="LineWidthPlus">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="LineWidthMinus">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>640</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Screenshots</string>
    </property>
    <widget class="QPushButton" name="MakeJpeg">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>.jpeg</string>
     </property>
    </widget>
    <widget class="QPushButton" name="MakeBmp">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>.bmp</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="MakeGifButt">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>730</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>32</height>
     </size>
    </property>
    <property name="text">
     <string>.gif</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>720</y>
      <width>101</width>
      <height>66</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="PlayButton">
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="StopButton">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyOpenGLWidget</class>
   <extends>QWidget</extends>
   <header location="global">Viwer/myopenglwidget.h</header>
   <container>1</container>
   <slots>
    <slot>changeZ()</slot>
    <slot>OpenModel()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>PlayButton</sender>
   <signal>clicked()</signal>
   <receiver>widget</receiver>
   <slot>changeZ()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>646</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
